src/components/usersettings/usersettingsbuilder.js

```diff
@@ -73,10 +70,7 @@ define(['appSettings', 'events'], function (appsettings, events) {
     UserSettings.prototype.get = function (name, enableOnServer) {
         var userId = this.currentUserId;
         if (!userId) {
-            // TODO: I'd like to continue to throw this exception but it causes issues with offline use
-            // Revisit in the future and restore it
-            return null;
-            //throw new Error('userId cannot be null');
+            throw new Error('userId cannot be null');
```

Comment:
why?

Reply:
We don't have an offline mode and it might be good to actually catch this error.

Reply:
We should just remove this entirely.

Reply:
Do you mean the error? I think it's right though, the API call will fail without an ID.

Reply:
yha